suite-native/module-device-onboarding/src/components/wallet-backup/WalletBackupSheet.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+
+import * as Haptics from 'expo-haptics';
+
+import { BottomSheet, Button, VStack } from '@suite-native/atoms';
+import { Icon } from '@suite-native/icons';
+import { Translation, useTranslate } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { WalletBackupCard } from './WalletBackupCard/WalletBackupCard';
+import { WalletBackupSheetFooter } from './WalletBackupSheetFooter';
+
+const containerStyle = prepareNativeStyle(utils => ({
+    marginBottom: utils.spacings.sp32,
+}));
+
+const legacyButtonStyle = prepareNativeStyle(utils => ({
+    alignSelf: 'center',
+    marginTop: utils.spacings.sp32,
+}));
+
+interface WalletBackupSheetProps {
+    isDisplayed: boolean;
+    closeModal: () => void;
+}
+
+export type WalletBackupType = 'shamir-single' | 'shamir-advanced' | '12-words' | '24-words';
+
+const walletOptions: WalletBackupType[] = [
+    'shamir-single',
+    'shamir-advanced',
+    '12-words',
+    '24-words',
+];
```

Comment:
nit:

```suggestion
const walletOptions = [
    'shamir-single',
    'shamir-advanced',
    '12-words',
    '24-words',
] as const satisfies WalletBackupType[];
```

Like that none can change the value of this list (it should be never changed by design).

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/614ee4b4f118a0e31c642a2014823012d4b420ae